### DISCUSSION TITLE
Pop up a prompt instead of using comments

### DISCUSSION THREAD

**Discussion by villesau** [community/community#41359] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Comments are very cumbersome way to ask Copilot to ask to do stuff. I would much rather have some nordic keyboard friendly hotkey (such as `option + shift + L` or `option + shift + i`) to open prompt and write what I want to happen. Any plans for supporting that in Jetbrains IDEs?

**Comment by iamcjwilliams** [comment]
@villesau Thank you for your feedback, our copilot team will be notified!

**Comment by alikor** [comment]
would really like this ability when refactoring or changing an existing block of code

**Comment by 0xNevo** [comment]
As an AI language model, I don't have specific information about JetBrains IDE product development roadmap or plans. However, if you are looking for a more efficient way to interact with Copilot within your Jetbrains IDEs, you may want to consider creating custom keybindings in your IDE settings.

Here are the steps to create a new keybinding:

Open your IDE settings (Preferences on Mac or Settings on PC).
Navigate to Keymap.
Search for the action you want to bind (such as "Open Copilot Prompt").
Right-click on the action and select "Add Keyboard Shortcut".
Press the keys combination you want to use as your hotkey (such as option + shift + L or option + shift + i)
Click OK to save.
You should now be able to use your custom hotkey to open the Copilot prompt in your JetBrains IDE.

Note that this functionality is dependent on whether Copilot is integrated within the IDE as a plugin or extension. If it is not currently available, you may want to reach out to the JetBrains support team or plugin developers to request this feature.
